addons/html_builder/static/src/sidebar/block_tab.js

```diff
@@ -288,9 +318,14 @@ export class BlockTab extends Component {
                 this.cancelSnippetPreview();
                 delete this.cancelSnippetPreview;
 
+                this.dispatchTo("on_snippet_out_dropzone_handlers", {
```

Comment:
who is using all this ressources ? on_snippet_over_dropzone_handlers, on_snippet_preview_handlers, on_snippet_over_dropzone_handlers

Reply:
For now, no one, but I made them to replace the hooks that were supposed to be overridden by mass_mailing (as resources are better) 
And I know that some future tasks will need them too, and it is to have the parallel with the drag and drop from the page too :slightly_smiling_face: